webextensions/javascript-apis.json

```diff
@@ -3081,6 +3081,27 @@
               }
             }
           }
+        },
+        "CSSOrigin": {
+          "__compat": {
+            "support": {
+              "chrome": {
+                "version_added": "false"
```

Comment:
`false` not `"false"`

Reply:
Fixed. Sorry, it seems the validation didn't care. :/

Reply:
Yes, we don't validate version numbers yet, it gets a bit complicated.